interpreter/src/main/java/org/qbicc/interpreter/impl/VmImpl.java

```diff
@@ -444,7 +444,7 @@ public void initialize() {
             // VMHelpers
             VmClassImpl vmHelpersClass = bootstrapClassLoader.loadClass("org/qbicc/runtime/main/VMHelpers");
 
-            vmHelpersClass.registerInvokable("get_class", (thread, target, args) -> ((VmObjectImpl) args.get(0)).getVmClass());
+            vmHelpersClass.registerInvokable("getClass", (thread, target, args) -> ((VmObjectImpl) args.get(0)).getVmClass());
```

Comment:
I think this will likely conflict with `Object.getClass()` - if you provide the parameter count argument though, it should be enough to disambiguate.

Reply:
Done.